ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-presubmits.yaml

```diff
@@ -111,3 +111,116 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - release-next
+    context: ci/prow/vsphere-e2e-vsphere-ocp-42
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      ci-operator.openshift.io/variant: vsphere
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-knative-serving-release-next-vsphere-e2e-vsphere-ocp-42
+    path_alias: knative.dev/serving
+    rerun_command: /test vsphere-e2e-vsphere-ocp-42
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-vsphere-ocp-42-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-vsphere-ocp-42
+        - --template=/usr/local/e2e-vsphere-ocp-42
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: vsphere
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-knative-serving-release-next__vsphere.yaml
+              name: ci-operator-misc-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-vsphere-ocp-42
+        - name: TEST_COMMAND
+          value: ./openshift/e2e-tests-openshift.sh
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-vsphere-ocp-42-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-vsphere-ocp-42
+          name: job-definition
+          subPath: cluster-launch-installer-upi-e2e.yaml
```

Comment:
I'm not sure this whole thing will work because of this template. For other Knative tests we use https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml which uses this image to execute our tests: https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml#L15 . It needs to actually include our source. The upi-e2e.yaml template seems to use a different image and I suppose it will only include OpenShift test suite, not Knative sources/test suites. Better to check with @stevekuznetsov or @petr-muller

Reply:
And I think the error in the CI confirms my concerns...

Reply:
I am not familiar with the `cluster-launch-installer-upi-e2e` template, but what @mgencur says is correct - it would be probably a good idea to get in touch with whoever wrote/owns that template and check if it's able to do what you expect to do.